src/components/Modals/NewCourse.vue

```diff
@@ -12,14 +12,16 @@ import coursesJSON from '../../assets/courses/courses.json';
 
 export default {
   props: {
-    semesterID: Number
+    isOnboard: Boolean,
+    semesterID: Number,
+    placeholderText: String
   },
   computed: {
     text() {
       return 'Search or Create New Course';
     },
     placeholder() {
-      return '"CS 1110", "Multivariable Calculus", etc.';
```

Comment:
Appreciate factoring this out, but now the old placeholder does not show up when adding a course normally. Make sure to pass it in!